ingestion/src/lib.rs

```diff
@@ -73,6 +79,8 @@ struct IngestorState {
     db_pool: PgConnectionPool,
     param_conversions: ParamConversions, // converts param codes to element ids
     permit_tables: Arc<RwLock<(ParamPermitTable, StationPermitTable)>>,
+    rove_connector: Arc<rove_connector::Connector>,
+    qc_pipelines: Arc<HashMap<(i32, RelativeDuration), rove::Pipeline>>,
```

Comment:
Somehow I feel that grasping the idea of pipelines is a key to understanding how rove works.

Reply:
Yes, I would say so.